examples/ps-slurm.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: parallelstore-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: parallelstore-slurm
+  region: us-east4
+  zone: us-east4-b
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network
+    source: modules/network/vpc
+    settings:
+      mtu: 8896
+
+  - id: psa_connect
+    source: community/modules/network/private-service-access
+    use: [network]
+    settings:
+      prefix_length: 24
+
+  - id: ps_instance
+    source: modules/file-system/parallelstore
+    use: [network, psa_connect]
+    settings:
+      local_mount: /ps
+
+  # Connect parallelstore instance with Compute VM running debian 12.
+  - id: compute_instances
+    source: modules/compute/vm-instance
+    use: [network, ps_instance]
+    settings:
+      instance_count: 1
+      instance_image:
+        family: debian-12
+        project: debian-cloud
+      machine_type: c2d-standard-112
+
+  # Connect parallelstore instance with slurm cluster.
+  - id: debug_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
+    settings:
+      node_count_dynamic_max: 4
+      machine_type: n2-standard-2
```

Comment:
What is a good machine to test with parallelstore. Can you really achieve full bandwidth with 1 core (2vcpu). I don't really know but seems like if you need bigger machine to test performance then we should put it in the blueprint. DAOS example also has c2 partition.

Reply:
From the Parallelstore team, at least c2-standard-4 is recommended. I put c2-s-60. LMKWYT.

Reply:
SGTM